src/dug/core/annotators/_base.py

```diff
@@ -40,12 +51,12 @@ def id_type(self):
         return self.id.split(":")[0]
 
     def add_search_text(self, text):
-        # Add text only if it's unique and if not empty string
+        "Add text only if it's unique and if not empty string"
```

Comment:
shouldn't it be commented out?

Reply:
It's a docstring: https://peps.python.org/pep-0257/

Docstrings are strongly encouraged by the PEP8 style guide: https://peps.python.org/pep-0008/

Reply:
ok, single quoted strings are not common